weblogic/README.md

```diff
@@ -0,0 +1,145 @@
+# Agent Check: weblogic
```

Comment:
```suggestion
# Agent Check: WebLogic
```

I think this happens when creating the integration (should be using the Display Name).